github-action/src/main.ts

```diff
@@ -138,6 +138,54 @@ export async function runMain(): Promise<void> {
 			return;
 		}
 
+		// If we have a platform specified and the image was built, get the image digest
+		if (buildResult.outcome === 'success') {
+			// Create a digests object to track digests for each platform
+			const digestsObj: Record<string, string> = {};
+			
+			if (platform) {
+				// Extract the image digest from the build output
+				if (buildResult.imageDigest) {
```

Comment:
I came across your PR tonight as I was trying to do multi-arch builds for my project. Looks like you have errors here between lines 148 and 150. It's `imageDigests` and not `imageDigest`.